iso8859-1/src/lib.rs

```diff
@@ -139,7 +139,7 @@ impl IsoLatin1Char {
     /// TODO
     /// ```
     pub fn is_numeric(&self) -> bool {
-        todo!()
+        self >= &IsoLatin1Char(0x30) && self <= &IsoLatin1Char(0x39)
```

Comment:
This implementation is incomplete. These standards don't specify the classification, but the Unicode standard specifies such classification for the value. Since those classifications and methods are very convenient, we can use Unicode classification for numbers to classify the bytes representing these Unicode values. That means that superscripts like "²" and fractions like "½" are also numeric digits.